templates/Deepcell_Postprocessing.ipynb

```diff
@@ -14,12 +14,16 @@
    "outputs": [],
```

Comment:
<p>So, the random images don't need to be generated everytime within the notebook.  They can be generated once, saved as images, and then loaded in like normal fovs.  That way, theres no random-image generating code in the notebook.</p><p> </p><p>Just use this code to generate some images, save them as .tiff's, and replace this code with data/image-loading code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>